### DISCUSSION TITLE
Where/how to report AMDGPU kernel-driver issues?

### DISCUSSION THREAD

**Discussion by lejeczek** [ROCm/ROCm#2965] [General]
Hi guys.
Where/how to report AMDGPU kernel-driver issues?
many thanks, L.

**Comment by kentrussell** [comment]
If it's something in the amdkfd code, you can use https://github.com/ROCm/ROCK-Kernel-Driver/issues . For amdgpu issues (install, graphics, etc) then the best place is https://gitlab.com/groups/amdgpu/-/issues . What kind of issue are you seeing? I can help steer you to the right spot
